### ISSUE TITLE
TO DO before March 1st

### ISSUE THREAD

**Issue by katieb1** [katieb1/NEvolve#3]
- [x] Profile conversion script and generate flame graph
- [x] Create generation script to run simulations (with ms) and create metadata file
* Generate more training data
* Figure out way to randomize simulations so we get a random collection of chunks / batch
* Change labels to categorical in ms
- [x] Get RAPIDS / PyTorch singularity image set up on respective clusters
- [ ] Work on parallelizing hyperparameter tuning script
- [ ] Create instructions file for chunking / data loader
- [ ] Create instructions file for setting up RAPIDS / PyTorch singularity image

**Comment by katieb1** [comment]
obsolete now
